trigger/task.go

```diff
@@ -328,7 +328,23 @@ func (t *taskTriggers) generate(sub *event.Subscription, pastTenseOverride, test
 	var payload interface{}
 	if sub.Subscriber.Type == event.JIRAIssueSubscriberType {
 		// We avoid creating BFG ticket in the case that the task is stranded to reduce noise for the Build Baron
-		if t.task.Details.Description == evergreen.TaskDescriptionStranded {
+		// If task is display, we skip ticket creation if all execution task failures are only 'stranded'
+		shouldSkipTicket := false
+		if t.task.DisplayOnly {
+			for _, executionTask := range t.task.ExecutionTasksFull {
```

Comment:
Sorry that this is confusing. UI added a few fields to the Task struct that are really only used for the UI / aren't stored in the database and should be removed (discussing this with Mohamed). Instead you'll need to get all execution tasks from the DB and then go through the loop (can use WithFields to just get the details though since that's all you need here).